packages/components/psammead-media-player/src/Canonical/index.jsx

```diff
@@ -17,29 +26,44 @@ const Canonical = ({ src, title, placeholderSrc }) => {
     top: 0;
     width: 100%;
     height: 100%;
-    ${placeholderSrc ? backgroundStyle : null}
+    ${showPlaceholder ? backgroundStyle : null}
   `;
 
   return (
-    <StyledIframe
-      src={src}
-      title={title}
-      allow="autoplay; fullscreen"
-      scrolling="no"
-      gesture="media"
-      allowFullScreen
-    />
+    <>
+      <StyledIframe
+        src={src}
+        title={title}
+        allow="autoplay; fullscreen"
+        scrolling="no"
+        gesture="media"
+        allowFullScreen
+      />
+      <noscript>
+        <MediaMessage
+          service={service}
+          noJsMessage={noJsMessage}
+          placeholderSrc={placeholderSrc}
+          placeholderSrcset={placeholderSrcset}
+        />
+      </noscript>
+    </>
   );
 };
 
 Canonical.propTypes = {
   src: string.isRequired,
+  placeholderSrcset: string,
```

Comment:
We'll likely want a README update if we're adding or changing props - https://www.npmjs.com/package/@bbc/psammead-media-player

Reply:
Done :)